File: app/src/main/java/yterletskyi/alias/StartPresenter.java
package yterletskyi.alias;

/**
 * Created by yterletskyi on 09.11.16.
 */

public class StartPresenter {

    private StartView mView;

    public StartPresenter(StartView view) {
        mView = view;
    }

    public void onGameBtnClicked() {
        mView.showGame();
    }

    public void onSettingsBtnClicked() {
        mView.showSettings();
    }




}

File: app/src/main/java/yterletskyi/alias/StartPresenter.kt
package yterletskyi.alias

/**
 * Created by yterletskyi on 09.11.16.
 */

class StartPresenter(private val mView: StartView) {

    fun onGameBtnClicked() {
        mView.showGame()
    }

    fun onSettingsBtnClicked() {
        mView.showSettings()
    }


}
